landlab/components/gravel_river_transporter/gravel_river_transporter.py

```diff
@@ -0,0 +1,484 @@
+import numpy as np
+from landlab import Component, HexModelGrid
+from landlab.grid.diagonals import DiagonalsMixIn
+from scipy.sparse.linalg import spsolve
+
+_ONE_SIXTH = 1.0 / 6.0
```

Comment:
This can be defined as self._ONE_SIXTH constant within the class (similar as line 152). If this is changed, please also update line 441.

Reply:
don